### ISSUE TITLE
Authorizations in Assistify request and expertise

### ISSUE THREAD

**Issue by mrsimpson** [assistify/Rocket.Chat#40]
_From @mrsimpson on March 29, 2017 6:51_

Currently, the ability to create expertises depends on being member of the expert channel. 
This is of course far from safe. 
During the implementation of the prototype, it has been tried to implement it properly, but [commenting out this line](https://github.com/mrsimpson/Rocket.Chat/blob/master/packages/assistify-help-request/startup/rolesAndPermissions.js#L50) leads to `streamer exceptions` server side at runtime.
```
I20170408-22:30:55.867(2)? Exception from sub stream-notify-room id X4WxQaL6z2dJJMP4B TypeError: Cannot read property 'indexOf' of undefined
I20170408-22:30:55.878(2)?   at [object Object].<anonymous> (/Users/jaegle/Projects/Rocket.Chat/.meteor/local/build/programs/server/packages/rocketchat_lib.js:3890:28)
I20170408-22:30:55.879(2)?   at Streamer.isReadAllowed (/Users/jaegle/Projects/Rocket.Chat/.meteor/local/build/programs/server/packages/rocketchat_streamer.js:343:38)
I20170408-22:30:55.881(2)?   at [object Object].<anonymous> (/Users/jaegle/Projects/Rocket.Chat/.meteor/local/build/programs/server/packages/rocketchat_streamer.js:478:16)
I20170408-22:30:55.882(2)?   at [object Object].<anonymous> (/Users/jaegle/Projects/Rocket.Chat/.meteor/local/build/programs/server/packages/rocketchat_lib.js:1114:15)
I20170408-22:30:55.882(2)?   at packages/matb33_collection-hooks.js:307:21
```
There has to be some misunderstanding of the API...

_Copied from original issue: mrsimpson/Rocket.Chat#11_

**Comment by mrsimpson** [comment]
Also changing to scope "Users" which is most probably appropriate did not change this.

**Comment by mrsimpson** [comment]
@graywolf336 Maybe you can help on that as well

**Comment by mrsimpson** [comment]
Has been included in 3df84aa7b02a124778d38d75fc11166d26f843c7
